src/runmode-dpdk.c

```diff
@@ -1337,6 +1363,10 @@ static void *ParseDpdkConfigAndConfigureDevice(const char *iface)
     // This counter is increased by worker threads that individually pick queue IDs.
     SC_ATOMIC_RESET(iconf->queue_id);
     SC_ATOMIC_RESET(iconf->inconsitent_numa_cnt);
+
+    // initialize LiveDev DPDK values
+    LiveDevice *ldev_instance = LiveGetDevice(iface);
```

Comment:
can this return NULL?

Reply:
It can but then it means that something is very off. Added an error check.